packages/contracts/src/resources/site.ts

```diff
@@ -158,11 +168,32 @@ export const siteApi = contract.router(
 				},
 			},
 		},
+		searchUsers: {
+			path: "/searchUsers",
+			method: "GET",
+			summary: "Get a list of matching users for autocomplete",
+			description:
+				"Get a list of users matching the provided query. Used for rendering suggestions in an autocomplete input for selecting users.",
+			query: z.object({
+				communityId: communitiesIdSchema,
+				email: z.string(),
+				name: z.string().optional(),
+				limit: z.number().optional(),
+			}),
+			responses: {
+				200: safeUserSchema
+					.extend({ member: communityMembershipsSchema.nullable().optional() })
+					.array(),
+			},
+		},
 	},
 	{
 		pathPrefix: "/api/v0/c/:communitySlug/site",
 		baseHeaders: z.object({
-			authorization: z.string().regex(/^Bearer /),
+			authorization: z
+				.string()
+				.regex(/^Bearer /)
+				.optional(),
```

Comment:
I'm not sure if making this optional was the right call, LMK if that breaks something. I assume #740 will fix it either way!

Reply:
ideally i'd do something like `z.union([authorization, cookies])` but that's not allowed unfortunately